Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -146,17 +146,17 @@ public enum ErrorId {
       @"
 No Dafny modifier, such as [`abstract`, `static`, `ghost`](https://dafny.org/latest/DafnyRef/DafnyRef#sec-declaration-modifiers) may be repeated
 Such repetition would be superfluous even if allowed.
-", Remove(true, "remove duplicate modifier"));
+".TrimStart(), Remove(true, "remove duplicate modifier"));
```

Comment:
Would it make sense to factor all these `TrimStart()` in the method Add itself?

Reply:
I considered that and it kinda hurt not doing that, but I feel if the caller decides to build a string using this @" followed by a newline pattern, then they should do the trim as well.

Reply:
Fair enough. I was a big user of Scala's `.trim()` feature so that the responsibility of the string was entirely the caller's site, so I won't complain. It's just that if it's literally used everywhere, I wouldn't feel bad if the called method would consider doing a bit of trimming instead of the argument.